pygsp/tests/test_graphs.py

```diff
@@ -321,3 +324,160 @@ def test_grid2dimgpatches(self):
+            self.assertEqual(e1.source(), e2.source())
+            self.assertEqual(e1.target(), e2.target())
+        for v1, v2 in zip(graph_gt.vertices(), graph2_gt.vertices()):
+            self.assertEqual(v1, v2)
+
+    def test_networkx_signal_export(self):
+        graph = graphs.BarabasiAlbert(N=100, seed=42)
+        np.random.seed(42)
+        signal1 = np.random.random(graph.N)
+        signal2 = np.random.random(graph.N)
+        graph.set_signal(signal1, "signal1")
+        graph.set_signal(signal2, "signal2")
+        graph_nx = graph.to_networkx()
+        for i in range(graph.n_nodes):
+            self.assertEqual(graph_nx.node[i]["signal1"], signal1[i])
+            self.assertEqual(graph_nx.node[i]["signal2"], signal2[i])
+
+    def test_graphtool_signal_export(self):
+        g = graphs.Logo()
+        np.random.seed(42)
+        s = np.random.random(g.N)
+        s2 = np.random.random(g.N)
+        g.set_signal(s, "signal1")
+        g.set_signal(s2, "signal2")
+        g_gt = g.to_graphtool()
+        # Check the signals on all nodes
+        for i, v in enumerate(g_gt.vertices()):
+            self.assertEqual(g_gt.vertex_properties["signal1"][v], s[i])
+            self.assertEqual(g_gt.vertex_properties["signal2"][v], s2[i])
+
+    def test_graphtool_signal_import(self):
+        g_gt = gt.Graph()
+        g_gt.add_vertex(10)
+
+        g_gt.add_edge(g_gt.vertex(3), g_gt.vertex(6))
+        g_gt.add_edge(g_gt.vertex(4), g_gt.vertex(6))
+        g_gt.add_edge(g_gt.vertex(7), g_gt.vertex(2))
+
+        vprop_double = g_gt.new_vertex_property("double")
+
+        vprop_double[g_gt.vertex(0)] = 5
+        vprop_double[g_gt.vertex(1)] = -3
+        vprop_double[g_gt.vertex(2)] = 2.4
+
+        g_gt.vertex_properties["signal"] = vprop_double
+        g = graphs.Graph.from_graphtool(g_gt, signals_names=["signal"])
+        self.assertEqual(g.signals["signal"][0], 5.0)
+        self.assertEqual(g.signals["signal"][1], -3.0)
+        self.assertEqual(g.signals["signal"][2], 2.4)
+
+    def test_networkx_signal_import(self):
+        g_nx = nx.Graph()
+        g_nx.add_edge(3, 4)
+        g_nx.add_edge(2, 4)
+        g_nx.add_edge(3, 5)
+        print(list(g_nx.node)[0])
+        dic_signal = {
+            2: 4.0,
+            3: 5.0,
+            4: 3.3,
+            5: 2.3
+        }
+
+        nx.set_node_attributes(g_nx, dic_signal, "signal1")
+        g = graphs.Graph.from_networkx(g_nx, signals_name=["signal1"])
+
+        for i, node in enumerate(g_nx.node):
+            self.assertEqual(g.signals["signal1"][i],
+                             nx.get_node_attributes(g_nx, "signal1")[node])
+
+    def test_save_load(self):
+        g = graphs.Bunny()
+        g.save("bunny.gml")
+        g2 = graphs.Graph.load("bunny.gml")
+        np.testing.assert_array_equal(g.W.todense(), g2.W.todense())
+
+
+suite = unittest.TestLoader().loadTestsFromTestCase(TestCaseImportExport)
```

Comment:
Here, you're overloading the `suite` defined at [line 326](https://github.com/epfl-lts2/pygsp/pull/32/files#diff-a413452692538643110c59a5b4fe01adR326). Hence, the `TestCase` with all the graph tests is not run anymore! Ẁe need to concatenate both suites, as follows:
```
suite_graphs = unittest.TestLoader().loadTestsFromTestCase(TestCase)
suite_io = unittest.TestLoader().loadTestsFromTestCase(ImportExport)
suite = unittest.TestSuite([suite_graphs, suite_io])
```

That will allow you to run the I/O test suite with `python -m unittest pygsp.tests.test_graphs.suite_io`.